releases/v20.1.1.md

```diff
@@ -0,0 +1,229 @@
+- Fixed a bug in which `(tuple).*` was only expanded to the first column in the tuple and the remaining elements were dropped. [#48290][#48290]
+- Fixed case where `PARTITION BY` and `ORDER BY` columns in window specifications were losing qualifications when used inside views. [#47715][#47715]
+- CockroachDB will no longer display a severe `internal error` upon certain privilege check failures via `pg_catalog` built-in functions. [#48242][#48242]
+- Fixed a bug where a read operation in a transaction with a past savepoint rollback would give an internal error for exceeding the maximum count of results requested [#48165][#48165]
+- Fixed a bug that could lead to data corruption or data loss if a replica was both the source of a snapshot and was being concurrently removed from the range and certain specific conditions exist inside RocksDB. This scenario is rare, but possible. [#48321][#48321]
+- The distinction between delete jobs for columns and dependent jobs for deleting indices, views and sequences is now better defined. [#48259][#48259]
+- Fixed incorrect results that could occur when casting negative intervals or timestamps to type `decimal`. [#48345][#48345]
+- Fixed an error that occurred when statistics collection was explicitly requested on a column with type `array`. [#48343][#48343]
+- Fixed a nil pointer dereference in Pebble's block cache due to a rare "double free" of a block. [#48346][#48346]
+- Fixed Pebble to properly mark `sstables` for compaction which contain range tombstones. This matches the behavior when using RocksDB and ensures that space used for temporary storage is reclaimed quickly. [#48346][#48346]
+- Fixed a bug introduced in v20.1 that could cause multiple index GC jobs to be created for the same schema change in rare cases. [#47818][#47818]
+- Fixed a bug where CockroachDB could return an internal error when performing a query with `CASE`, `AND`, `OR` operators in some cases when it was executed via the vectorized engine. [#48072][#48072]
+- TLS 1.3 is now disabled by default to improve compatibility with Java. [#48365][#48365]
+- Fixed a rare bug where stats were not automatically generated for a new table. [#48027][#48027]
+- Fixed a panic that could occur when `SHOW RANGES` or `SHOW RANGE FOR ROW` was called with a virtual table. [#48347][#48347]
+- Made SRV resolution non-fatal for join list records to align with the standard and improve reliability of node startup. [#48349][#48349]
+- Fixed a rare bug causing a range to deadlock and all the writes to the respective range to timeout. [#48303][#48303]
+- Fixed a long-standing bug where HTTP requests would start to fail with error 503 "`transport: authentication handshake failed: io: read/write on closed pipe`" and never become possible again until the node is restarted. [#48456][#48456]
+- Fixed a bug where vectorized queries on composite datatypes could sometimes return invalid data. [#48463][#48463]
```

Comment:
I would also move this up.  Anything about data being incorrect/invalid/corrupted/etc. is something I would want to see first and foremost as a user.